tests/cases/whitespace/use-namespace

```diff
@@ -11,6 +11,13 @@ namespace Vendor;
 
 new Foo();
 
----MESSAGES---
-5:1 PSR2.Namespaces.UseDeclaration.UseBeforeNamespace
+---FIXED---
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor;
+
+new Foo();
```

Comment:
This is wrong, as the `use` statement is being removed. One of the new sniffs must be misbehaving.

Reply:
Looking at it, `UseFromSameNamespace` is actually correct because the `use` statement is in the global namespace, where it's not needed. Would still rather `UseBeforeNamespace` is triggered.

Reply:
Can use a namespaced class to get round this for now.

Reply:
(Just checked what happens and the code's going to be broken either way.)